stock_available_mrp/tests/test_potential_qty.py

```diff
@@ -454,6 +454,42 @@ def test_potential_qty_list(self):
         )
 
         self.assertEqual(
-            {p1.id: 3.0, p2.id: 3.0, p3.id: 0.0},
+            {p1.id: -1.0, p2.id: 3.0, p3.id: 0.0},
+            {p.id: p.potential_qty for p in products}
+        )
+
+    def test_potential_quantity_service_and_stockable(self):
+        p1 = self.product_model.create({'name': 'Test Product', 'type': 'product'})
+        p2 = self.product_model.create({'name': 'Test PService', 'type': 'service'})
+
+        # P1 need one P2
+        self.create_simple_bom(p1, p2)
+
+        self.product_model.invalidate_cache()
+
+        products = self.product_model.search(
+            [('id', 'in', [p1.id, p2.id])]
+        )
+
+        self.assertEqual(
+            {p1.id: -1.0, p2.id: 0.0},
+            {p.id: p.potential_qty for p in products}
+        )
```

Comment:
NItpicking, but IMO you can simply do:
```suggestion
        self.assertEqual(stockable_product.potential_qty, -1.0)
```
No need to check the potential qty of a service product...